extensions/csharp/extension.toml

```diff
@@ -1,7 +1,7 @@
 id = "csharp"
 name = "C#"
 description = "C# support."
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
Can we remove the version bump from this PR?

We bump the versions separately.

Reply:
Done.